### ISSUE TITLE
Empty sections in https://hired.com/library

### ISSUE THREAD

**Issue by rstormsf** [fortitude/fortitude-docs#11]
https://hired.com/library#logo
https://hired.com/library#pagination
https://hired.com/library#loader

there is nothing in there.

**Comment by lifeiscontent** [comment]
@rstormsf you're correct, I put them there as placeholders because they will be there in the coming days/weeks. I wanted them to stick out to the rest of the team so that maybe someone else would do them for me :stuck_out_tongue_closed_eyes:
